linera-rpc/examples/generate-format.rs

```diff
@@ -63,7 +63,8 @@ enum Action {
 #[derive(clap::Parser, Debug)]
 #[clap(
     name = "Format generator",
-    about = "Trace serde (de)serialization to generate format descriptions"
+    about = "Trace serde (de)serialization to generate format descriptions",
+    version = clap::crate_version!(),
```

Comment:
Ok so we will need to keep adding this one, otherwise the CLI wrapper won't work (which you probably observed). Maybe add note in the coding guidelines?

Reply:
There was a note already, lol

Reply:
I've updated the note to be more specific about the `version` annotation that's required.